wolfcrypt/src/aes.c

```diff
@@ -4281,28 +4279,108 @@ int wc_AesGcmEncrypt(Aes* aes, byte* out, const byte* in, word32 sz,
                    byte* authTag, word32 authTagSz,
                    const byte* authIn, word32 authInSz)
 {
-#if defined(FREESCALE_LTC_AES_GCM)
+#if defined(FREESCALE_LTC_AES_GCM) ||((defined(STM32F2_CRYPTO) || defined(STM32F4_CRYPTO)) && !defined(WOLFSSL_STM32_CUBEMX))
     byte *key;
     uint32_t keySize;
-    status_t status;
-
-    if (authTagSz < WOLFSSL_MIN_AUTH_TAG_SZ) {
```

Comment:
I'll revert this for Kojo. I agree it should not have been removed. Or only removed for STM32, but still doesn't make sense to me.